server/sonar-server/src/main/java/org/sonar/server/rule/RuleOperations.java

```diff
@@ -43,7 +43,8 @@
  * @deprecated to be dropped in 4.4
  */
 @Deprecated
-public class RuleOperations implements ServerComponent {
+@ServerSide
+public class RuleOperations {
```

Comment:
![INFO](https://raw.githubusercontent.com/henryju/image-hosting/master/info.png)Do not forget to remove this deprecated code someday.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)